docs/installation.md

```diff
@@ -23,7 +23,7 @@ git clone https://github.com/Netflix/dispatch-docker.git
 Before starting the installation, we strongly recommend you check out [how to configure your Dispatch instance](configuration/) as you'd need to rebuild your images \(`docker-compose build`\) if you want to change your configuration settings. You may copy and edit the example configs provided in the repository. If none exists, the install script will use these examples as actual configurations.
 
 {% hint style="info" %}
-Note: Dispatch will not start without at least a few required configuration variables. See the example [env](https://github.com/Netflix/dispatch/blob/develop/docker/.env.example).
+Note: Dispatch will not start without at least a few required configuration variables. See the example [env](https://github.com/Netflix/dispatch/blob/master/docker/.env.example).
```

Comment:
Let's use `latest` instead of `master`.

Reply:
Nice, didn't know that was an option :)

Reply:
Yeah, Github allows you to use branch names or tags. Latest points to the last commit of our most recent release. Thanks for the PR!